bot.py

```diff
@@ -29,6 +33,13 @@
 ROLE_ID = 1168850068665794661
 GUILD_ID = 1099793030678069338
 
+#OSCAR TESTING
+# CHANNEL_ID = 723899751732346964
+# ROLE_ID = 1180661716107931658
```

Comment:
Do we need this committed if it's specific to Oscar's local testing?

Reply:
ideally we have this in the dotenv like the bot token